index.html

```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SKYDRONE</title>
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <link rel="shortcut icon" href="/src/favicon.ico">
```

Comment:
Do we need this slash?
```suggestion
    <link rel="shortcut icon" href="src/favicon.ico">
```